### PR TITLE
Look for blueprint data in EDCD/EDDP

### PR DESCRIPTION
**Pull request by StivlWrith**
To be able to go to take the data in EDCD/EDDP is also to have the possibility to have updated datas

### CODE REVIEW

File: DataDefinitions/BlueprintMaterials.cs [hunk 1]
```diff
@@ -20,7 +20,7 @@ public BlueprintMaterials(List<MaterialAmount> materials)
 
         static BlueprintMaterials()
         {
-            string data = Net.DownloadString("http://api.eddp.co/_blueprintrequirements");
+            string data = Net.DownloadString("https://raw.githubusercontent.com/EDCD/EDDP/master/blueprintrequirements.json");
```
Comment (Tkael) [inline DataDefinitions/BlueprintMaterials.cs]: Please revise to use the server URL as specified in Constants.cs, like this:
`string data = Net.DownloadString(Constants.EDDI_SERVER_URL + "blueprintrequirements.json");`

File: MaterialMonitor/MaterialMonitor.cs [hunk 2]
```diff
@@ -381,7 +381,7 @@ private void readMaterials()
         private void populateMaterialBlueprints()
         {
             // TODO: Transfer this function to a new server
```
Comment (Tkael) [inline MaterialMonitor/MaterialMonitor.cs]: TODO comment can be deleted now.

File: MaterialMonitor/MaterialMonitor.cs [hunk 3]
```diff
@@ -381,7 +381,7 @@ private void readMaterials()
...
-            string data = Net.DownloadString("http://api.eddp.co/_materialuses");
+            string data = Net.DownloadString("https://raw.githubusercontent.com/EDCD/EDDP/master/materialuses.json");
```
Comment (Tkael) [inline MaterialMonitor/MaterialMonitor.cs]: `string data = Net.DownloadString(Constants.EDDI_SERVER_URL + "materialuses.json");`

File: MaterialMonitor/MaterialMonitor.cs [hunk 4]
```diff
@@ -399,7 +399,7 @@ private void populateMaterialBlueprints()
         private void populateMaterialLocations()
         {
             // TODO: Transfer this function to a new server
```
Comment (Tkael) [inline MaterialMonitor/MaterialMonitor.cs]: TODO comment can be deleted now.

File: MaterialMonitor/MaterialMonitor.cs [hunk 5]
```diff
@@ -399,7 +399,7 @@ private void populateMaterialBlueprints()
...
-            string data = Net.DownloadString("http://api.eddp.co/_materiallocations");
+            string data = Net.DownloadString("https://raw.githubusercontent.com/EDCD/EDDP/master/materiallocations.json");
```
Comment (Tkael) [inline MaterialMonitor/MaterialMonitor.cs]: `string data = Net.DownloadString(Constants.EDDI_SERVER_URL + "materiallocations.json");`

**Comment by StivlWrith** [comment]
changes applied as requested

**Pull request merged**